nuxeo-java-client-test/src/test/java/org/nuxeo/client/ITAuthentication.java

```diff
@@ -125,7 +126,8 @@ public void itReturnProperExceptionWhenLoginWithOauth2AndJwtBearer() {
                 serverVersion.majorVersion() == LTS_10_10.majorVersion()
                         && serverVersion.isGreaterThan(LTS_10_10.hotfix(63))
                         || serverVersion.majorVersion() == LTS_2021.majorVersion()
-                                && serverVersion.isGreaterThan(LTS_2021.minor(23)));
+                                && serverVersion.isGreaterThan(LTS_2021.minor(23))
+                        || serverVersion.majorVersion() == LTS_2023.majorVersion());
```

Comment:
Maybe we miss a `#isGreaterThanOrEquals` method here? This test is valide for all LTS 2023+.
WDYT @kevinleturc ?

Reply:
Indeed, we should use `isGreaterThanOrEquals`method only in order to get all LTS 2023 and future ones too.
```suggestion
                        || serverVersion. isGreaterThan(LTS_2023));
```